lib/Sema/UnresolvedNameTyMapper.hpp

```diff
@@ -30,15 +39,75 @@ class UnresolvedNameTyMapper
 
     void preVisitNamespaceDecl(glu::ast::NamespaceDecl *decl)
     {
+        TypeMappingVisitorBase::preVisitNamespaceDecl(decl);
         _scopeTable = _scopeTable->getLocalNamespace(decl->getName());
     }
 
     // overriding this is safe as there is no NODE_TYPEREF in NamespaceDecl
```

Comment:
you can remove this comment now that you're calling the superclass visit function